index.rst

```diff
@@ -0,0 +1,15 @@
+.. PPMS-PDFD documentation master file, created by
+   sphinx-quickstart on Tue Dec 12 14:22:39 2017.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Piwik PRO Developers documentation!
+=====================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Mobile SDK documentation
```

Comment:
I was trying to avoid setting captions for each section. I thought that maybe we could use them to separate apps, but only if it would be necessary. I would start without captions.

Reply:
I will leave this caption here as this is complete documentation for SDK section